packages/fxa-settings/src/components/ButtonDownloadRecoveryKey/index.tsx

```diff
@@ -0,0 +1,126 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { useAccount, useFtlMsgResolver } from '../../models';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { logViewEvent } from '../../lib/metrics';
+
+interface ButtonDownloadRecoveryKeyProps {
+  recoveryKeyValue: string;
+  viewName: string;
```

Comment:
does it feel silly to document the incoming viewNames with an enum or something like that? The thought occurred to me bc my eye snagged on the fact that the storybook file uses a different viewname than the test file but, but it does feel a little like overkill

Reply:
We could, but I'm not sure what the views will be called yet :) We could add those in as an enum later on when we start to use the button in pages?